packages/map/src/scss/editor-panel.scss

```diff
@@ -280,6 +280,9 @@
       span.h5 {
         font-size: 0.8em;
       }
+      & p {
```

Comment:
I think this should use a more focused selector. I'm not sure we want all paragraph items inside of accordion panels to have a margin top on them.